src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -6,43 +6,48 @@
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
     "tagged" : [ "friends" ],
-    "policies" : ["#P12345"]
+    "policies" : ["Policy_1111>null"]
```

Comment:
The string "null" could be "no_url" as the insurance agent may not know what is a null when checking the json file.

Reply:
Great suggestion! Thanks!